Library/Homebrew/locale.rb

```diff
@@ -68,6 +68,11 @@ def eql?(other)
   end
   alias == eql?
 
+  def detect(others)
+    others.detect { |other| eql?(*other) } ||
```

Comment:
No need for the `*` here.

Reply:
https://github.com/Homebrew/brew/pull/4224#discussion_r191364518

Reply:
`eql?` and `include?` are not implemented for multiple arguments. It should be `other.any? { |o| eql?(o) }`.

Reply:
Maybe rename `others` to `locale_groups` and `other` to `locales`.